app/src/main/java/org/liberty/android/fantastischmemo/ui/RecentListFragment.java

```diff
@@ -68,6 +69,7 @@
 
     private RecyclerView recentListRecyclerView;
     private RecentListAdapter recentListAdapter;
+    public static String SHUFFLE_Cards = "shufflecards";
```

Comment:
According to the standards, static field names start with s
ie: private static MyClass sSingleton;
if it was a final static then it'd be all caps
public static final int SOME_CONSTANT

Reply:
@j-suguma so sShuffleCards instead?

Reply:
I think it has to be SHUFFLE_CARDS

Reply:
caps is for final variables usually